civictechprojects/static/css/_vars.scss

```diff
@@ -0,0 +1,17 @@
+//pattern for variable names: generic-to-specific.
+//when possible name variables after function
+
+//colors
+$color-orange: #f79e02; // brand
+$color-green: #86c232;
+$color-green-dark: #61892f;
```

Comment:
At some point, we will want to move away from variable names the imply certain values (like "green") and use names that specify the styling role (like "primary-button-color", or like "color-text-light").  That's not for right now, though, since we do not have those styling roles specified.  Ideally, a designer would determine what those styling roles are based on patterns in the visual components.